### PR TITLE
Or 1734 add usdc bridge handler for thanos event listener

### PR DESCRIPTION
**Pull request by xxeonge**
- set TON's L1Token Address when L2 Dep/With Event

    -> In the dep/with event that occurs in L2StandardBridge, the Ton address appears as zeroAddress. This was intended to be expressed as TON's L1 address.
    ->  TON's address not found inside the event. Therefore, I used the method of retrieving from tokenInfoMap using the symbol "TON"
    
Thank you so much!

### CODE REVIEW

File: internal/app/thanos-notif/app.go [hunk 1]
```diff
@@ -229,13 +230,13 @@ func (app *App) L2DepEvent(vLog *types.Log) {
 
 	if isETH {
 		title = fmt.Sprintf("[" + app.cfg.Network + "] [ETH Deposit Finalized]")
-		text = fmt.Sprintf("Tx: "+app.cfg.L2ExplorerUrl+"/tx/%s\nFrom: "+app.cfg.L1ExplorerUrl+"/address/%s\nTo: "+app.cfg.L2ExplorerUrl+"/address/%s\nL1Token: ETH\nL2Token: "+app.cfg.L2ExplorerUrl+"/token/%s\nAmount: %+v%s", vLog.TxHash, l2Dep.From, l2Dep.To, l2Dep.L2Token, Amount, tokenSymbol)
+		text = fmt.Sprintf("Tx: "+app.cfg.L2ExplorerUrl+"/tx/%s\nFrom: "+app.cfg.L1ExplorerUrl+"/address/%s\nTo: "+app.cfg.L2ExplorerUrl+"/address/%s\nL1Token: ETH\nL2Token: "+app.cfg.L2ExplorerUrl+"/token/%s\nAmount: %+v %s", vLog.TxHash, l2Dep.From, l2Dep.To, l2Dep.L2Token, Amount, tokenSymbol)
 	} else if isTON {
 		title = fmt.Sprintf("[" + app.cfg.Network + "] [ERC-20 Deposit Finalized]")
```
Comment (ohbyeongmin) [inline internal/app/thanos-notif/app.go:235]: I think [NativeToken Deposit Finalized] or [TON Deposit Finalized] is better.

File: internal/app/thanos-notif/app.go [hunk 2]
```diff
@@ -294,13 +295,13 @@ func (app *App) L2WithEvent(vLog *types.Log) {
 
 	if isETH {
 		title = fmt.Sprintf("[" + app.cfg.Network + "] [ETH Withdrawal Initialized]")
-		text = fmt.Sprintf("Tx: "+app.cfg.L2ExplorerUrl+"/tx/%s\nFrom: "+app.cfg.L2ExplorerUrl+"/address/%s\nTo: "+app.cfg.L1ExplorerUrl+"/address/%s\nL1Token: ETH\nL2Token: "+app.cfg.L2ExplorerUrl+"/token/%s\nAmount: %+v%s", vLog.TxHash, l2With.From, l2With.To, l2With.L2Token, Amount, tokenSymbol)
+		text = fmt.Sprintf("Tx: "+app.cfg.L2ExplorerUrl+"/tx/%s\nFrom: "+app.cfg.L2ExplorerUrl+"/address/%s\nTo: "+app.cfg.L1ExplorerUrl+"/address/%s\nL1Token: ETH\nL2Token: "+app.cfg.L2ExplorerUrl+"/token/%s\nAmount: %+v %s", vLog.TxHash, l2With.From, l2With.To, l2With.L2Token, Amount, tokenSymbol)
 	} else if isTON {
 		title = fmt.Sprintf("[" + app.cfg.Network + "] [ERC-20 Withdrawal Initialized]")
```
Comment (ohbyeongmin) [inline internal/app/thanos-notif/app.go:300]: Too this.

File: internal/app/thanos-notif/app.go [hunk 3]
```diff
@@ -129,7 +130,7 @@ func (app *App) ERC20DepEvent(vLog *types.Log) {
 
 	// Slack notify title and text
 	title := fmt.Sprintf("[" + app.cfg.Network + "] [ERC-20 Deposit Initialized]")
-	text := fmt.Sprintf("Tx: "+app.cfg.L1ExplorerUrl+"/tx/%s\nFrom: "+app.cfg.L1ExplorerUrl+"/address/%s\nTo: "+app.cfg.L2ExplorerUrl+"/address/%s\nL1Token: "+app.cfg.L1ExplorerUrl+"/token/%s\nL2Token: "+app.cfg.L2ExplorerUrl+"/token/%s\nAmount: %+v%s", vLog.TxHash, erc20Dep.From, erc20Dep.To, erc20Dep.L1Token, erc20Dep.L2Token, Amount, tokenSymbol)
+	text := fmt.Sprintf("Tx: "+app.cfg.L1ExplorerUrl+"/tx/%s\nFrom: "+app.cfg.L1ExplorerUrl+"/address/%s\nTo: "+app.cfg.L2ExplorerUrl+"/address/%s\nL1Token: "+app.cfg.L1ExplorerUrl+"/token/%s\nL2Token: "+app.cfg.L2ExplorerUrl+"/token/%s\nAmount: %+v %s", vLog.TxHash, erc20Dep.From, erc20Dep.To, erc20Dep.L1Token, erc20Dep.L2Token, Amount, tokenSymbol)
```
Comment (ohbyeongmin) [inline internal/app/thanos-notif/app.go:133]: Amount format %+v -> %s

**Review by ohbyeongmin — COMMENTED** [review]
Also, USDC title is [USDC Deposit/Withdrawal Init/filnal] is better

**Comment by ohbyeongmin** [comment]
Since the format of alarm messages is generally similar, managing and reusing the formatting in one place would result in cleaner code and facilitate maintenance. 

For example:
```go
const (
  fmtTitle   = "[%s] [%s]"
  fmtTokenText = "Tx: %s/tx/%s\n" +
       "From: %s/address/%s\n" +
       "To: %s/address/%s\n" +
       "L1Token: %s/token/%s\n" +
       "L2Token: %s/token/%s\n" +
       "Amount: %s %s\n"
)

title := fmt.Sprintf(fmtTitle, app.cfg.Network, "ERC-20 Withdrawal Finalized")
text := fmt.Sprintf(fmtTokenText, app.cfg.L1ExplorerUrl, vLog.TxHash,
	app.cfg.L2ExplorerUrl, erc20With.From,
	app.cfg.L1ExplorerUrl, erc20With.To,
	app.cfg.L1ExplorerUrl, erc20With.L1Token,
	app.cfg.L2ExplorerUrl, erc20With.L2Token,
	Amount, tokenSymbol)
```
No need to make changes for this PR; just keep it in mind for future reference.

**Comment by xxeonge** [comment]
> 알람 메시지의 형식은 일반적으로 비슷하므로, 한 곳에서 형식을 관리하고 재사용하면 코드가 더 깔끔해지고 유지 관리가 용이해집니다.
> 
> 예를 들어:
> 
> ```go
> const (
>   fmtTitle   = "[%s] [%s]"
>   fmtTokenText = "Tx: %s/tx/%s\n" +
>        "From: %s/address/%s\n" +
>        "To: %s/address/%s\n" +
>        "L1Token: %s/token/%s\n" +
>        "L2Token: %s/token/%s\n" +
>        "Amount: %s %s\n"
> )
> 
> title := fmt.Sprintf(fmtTitle, app.cfg.Network, "ERC-20 Withdrawal Finalized")
> text := fmt.Sprintf(fmtTokenText, app.cfg.L1ExplorerUrl, vLog.TxHash,
> 	app.cfg.L2ExplorerUrl, erc20With.From,
> 	app.cfg.L1ExplorerUrl, erc20With.To,
> 	app.cfg.L1ExplorerUrl, erc20With.L1Token,
> 	app.cfg.L2ExplorerUrl, erc20With.L2Token,
> 	Amount, tokenSymbol)
> ```
> 
> 이 PR에 대해서는 변경할 필요가 없습니다. 향후 참고용으로만 기억해 두세요.

Thank you so much!!🙇‍♀️

**Review by ohbyeongmin — APPROVED** [review]
Thank you!

**Pull request merged**